cmake/modules/hunter_check_toolchain_definition.cmake

```diff
@@ -0,0 +1,68 @@
+# Copyright (c) 2016 NeroBurner
+# All rights reserved.
+
+include(CMakeParseArguments) # cmake_parse_arguments
+
+include(hunter_test_string_not_empty)
+
+function(hunter_check_toolchain_definition)
+  set(_function_name "hunter_check_toolchain_definition")
```

Comment:
There is no need to use `_*` for variables - we are inside function. Function has separate variable scope.